.circleci/config.yml

```diff
@@ -7,15 +7,15 @@ jobs:
     steps:
       - checkout
       - restore_cache:
-          key: dependency-cache-{{ checksum "package.json" }}
+          key: v1-dependency-cache-{{ checksum "package.json" }}
```

Comment:
Is there any chance this is better handled by `persist_to_workspace` so we're not constantly tweaking the cache? I mean the cache speeds up all builds where the package.json hasn't changed, but… Dunno, we're also changing the cache key every PR it seems like, right? <_<

Reply:
Based on some quick [googling](https://hackernoon.com/circleci-performance-difference-between-cache-and-workspace-5567679c3601), seems like `persist_to_workspace` offers better performance? Can we tackle that in another PR?

I only changed the cache key here to try to manually bust it. Looking at the [docs](https://circleci.com/docs/2.0/caching/#clearing-cache), it seemed like changing the name is the only way to rebuild with no cache.  I wanted a rebuild without cache even though the package.json didn't change, because of a wonky install of the react-scripts package. (...It still installed incorrectly, and I'm not sure why.) Otherwise I wouldn't have updated the cache key name.

Reply:
We can absolutely tackle it in another PR, that's what I was indicating when I approved this one sorry. And there's no rush at all to do it, we can choose not to, etc.

Basically, whatever works ;)

Merging this!